core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/deletevolumes/DeleteVolumesService.java

```diff
@@ -0,0 +1,70 @@
+package com.sequenceiq.cloudbreak.core.flow2.cluster.deletevolumes;
+
+import static com.sequenceiq.cloudbreak.event.ResourceEvent.CLUSTER_DELETEDVOLUMES;
+import static com.sequenceiq.cloudbreak.event.ResourceEvent.CLUSTER_DELETINGVOLUMES;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.Status;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.StackDeleteVolumesRequest;
+import com.sequenceiq.cloudbreak.cloud.CloudConnector;
+import com.sequenceiq.cloudbreak.cloud.context.AuthenticatedContext;
+import com.sequenceiq.cloudbreak.cloud.init.CloudPlatformConnectors;
+import com.sequenceiq.cloudbreak.cloud.model.CloudPlatformVariant;
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.model.Platform;
+import com.sequenceiq.cloudbreak.cloud.model.Variant;
+import com.sequenceiq.cloudbreak.core.flow2.stack.CloudbreakFlowMessageService;
+import com.sequenceiq.cloudbreak.domain.Resource;
+import com.sequenceiq.cloudbreak.dto.StackDto;
+import com.sequenceiq.cloudbreak.reactor.api.event.resource.DeleteVolumesRequest;
+import com.sequenceiq.cloudbreak.service.resource.ResourceService;
+import com.sequenceiq.common.api.type.ResourceType;
+
+@Service
+public class DeleteVolumesService {
+
+    @Inject
+    private CloudbreakFlowMessageService flowMessageService;
+
+    @Inject
+    private CloudPlatformConnectors cloudPlatformConnectors;
+
+    @Inject
+    private ResourceService resourceService;
+
+    public void deleteVolumes(StackDeleteVolumesRequest payload) {
+        flowMessageService.fireEventAndLog(payload.getStackId(),
+                Status.UPDATE_IN_PROGRESS.name(),
+                CLUSTER_DELETINGVOLUMES,
+                payload.getGroup(),
+                payload.getStackId().toString());
+    }
```

Comment:
method name is misleading since it's just sending event. Also, Do we even need this method here since there is no business logic involved? From action class itself, we can send the event which we are doing other cases.

Reply:
done